docs/self-hosted/install/install-on-aws-script.md

```diff
@@ -21,6 +21,7 @@ export AWS_SECRET_ACCESS_KEY=...
 
 ## 2. Run the installer image
 ```bash
+mkdir -p $PWD/awsinstall
```

Comment:
I don't think that's neccesary. Docker should create the folder if it doesn't exist.

Reply:
It does not on Linux/in Gitpod.